tools/chdbg/README.md

```diff
@@ -0,0 +1,28 @@
+# chdbg
```

Comment:
we should add some more information here on how to use it and read it. the below excerpt isn't very intuivutve since its hard to tell which store is the issue.

In the sdk we have a convention of providing a cli with helper text, should we adopt that here @julienrbrt ?

Reply:
Maybe all tools should use cobra yes.